traits/trait_converters.py

```diff
@@ -109,17 +109,23 @@ def trait_for(trait):
         return Trait(trait)
 
 
-def mapped_trait_for(trait):
+def mapped_trait_for(trait, name):
     """ Returns the 'mapped trait' definition for a mapped trait.
```

Comment:
It would be a good opportunity to improve the docstring here given that we know what `trait` and `name` are.

Reply:
This is a backwards compatibility break for anyone using `mapped_trait_for`. Please can you do a quick check that we don't have code outside traits using `mapped_trait_for`?

It's a bit of a grey area: I think it's okay to change this---it's reasonable to say that if you're importing from anything other than one of the `*.api` modules then you're on your own with respect to backwards compatibility breakage. But we should still make the check.

Also, agreed with @kitchoi that we should update the docstring. The traits stubs may also need updating.

Reply:
Couldn't find any other usage of `mapped_trait_for` on Github. 

Updated the docstring. `mapped_trait_for` signature isn't in traits stubs so no update there.